nat-lab/tests/telio.py

```diff
@@ -870,7 +885,7 @@ async def save_logs(self) -> None:
         os.makedirs(log_dir, exist_ok=True)
 
         log_content = await self.get_log()
-
+        network_info_info = await self.get_network_info()
```

Comment:
I think it would be a slight improvement by moving it nearby where the writing happens

Reply:
Done